components/Faq.vue

```diff
@@ -1,27 +1,124 @@
 <template>
-  <div class="Parent">
-    <h2 class="Parent-Heading">
-      <!-- Tab TBD -->
-    </h2>
-    <SideNavigationOverview
-      :categories="items"
-      :active-category="activeCategory"
-      @clicked="scrollToCategory"
-    />
-    <!-- Add Scroll Area if SideNavigationOverview is shown -->
-    <div
-      :class="{ 'FaqContent-Scroll-Area': !isSmallScreenWidth }"
-      @scroll.passive="debounce"
-    >
-      <div class="FaqContent-Scroll-Length" :style="{ height: scrollLength }">
-        <div
-          v-for="(item, i) in items"
-          :id="`faq-content-${i}`"
-          :key="i"
-          class="FaqContent-Wrapper"
-        >
-        </div>
+  <div class="Faq">
+    <div class="Faq-Header">Frequently Asked Questions</div>
+    <div class="Faq-Question1">
+      <div class="Faq-Question">Where does all this data come from?</div>
+      <div class="Faq-Expand">+</div>
+    </div>
+    <div class="Faq-Question2">
+      <div class="Faq-Question-Multi">
+        When are upcoming elections?
+        Which races have an impact on police spending, or policing in general?
       </div>
+      <div class="Faq-Expand">+</div>
+    </div>
+    <div class="Faq-Question3">
+      <div class="Faq-Question">Who can I call or write to voice my opinion?</div>
+      <div class="Faq-Expand">+</div>
     </div>
   </div>
 </template>
+
+<style lang="scss" scoped>
+@font-face {
+  font-family: "Avenir";
+  src: url("../assets/AvenirLTStd-Medium.otf");
+}
+.Faq {
+  position: absolute;
+  width: 1440px;
+  height: 497px;
+  left: 0px;
+  top: 936px;
+  background: #f1f8f8;
+  &-Header {
+    position: absolute;
+    width: 553px;
+    height: 33px;
+    left: 443px;
+    top: 64px;
+    font-family: Avenir;
+    font-style: normal;
+    font-weight: 800;
+    font-size: 24px;
+    line-height: 33px;
+    text-align: center;
+    color: #000000;
+  }
+  &-Question1 {
+    position: absolute;
+    width: 819px;
+    height: 84px;
+    left: 311px;
+    top: 137px;
+    background: #ffffff;
+    border: 1px solid #bdbdbd;
+    box-sizing: border-box;
+  }
+  &-Question2 {
+    position: absolute;
+    width: 819px;
+    height: 84px;
+    left: 311px;
+    top: 220px;
+    background: #ffffff;
```

Comment:
Can you move #ffffff to the variables.scss file and create a parameter for this color? This way we can reuse it across the variables stylings. If we decided to change the color we will only need to update one place to impact all usages.

Reply:
@jwiederhold done!